keps/sig-node/20190131-runas-groupid.md

```diff
@@ -0,0 +1,44 @@
+---
+kep-number: 0
+title: RunAsGroup support in PodSpec and PodSecurityPolicy
+authors:
+  - "@krmayankk"
+owning-sig: sig-node
+participating-sigs:
+  - sig-auth
+reviewers:
+  - "@tallclair"
+  - "@mrunalp"
+approvers:
+  - "@liggitt"
+  - "@derekwaynecarr"
+editor: TBD
+creation-date: 2017-06-21
+last-updated: 2017-09-14
+status: implementable
+---
+
+# RunAsGroup support in PodSpec and PodSecurityPolicy
+
+https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/runas-groupid.md
```

Comment:
Can you copy most of the content from the design proposal to here? Preferably as a separate commit. The KEP should be the source-of-truth for the design.